olatundesleek/capcoinadminfr
# #6 feature(auth):improve authentication

#### What does this PR do?
Improve authentication and authorization on the UI
#### Description of Task to be completed?
- implement logout
- restrict the user from the dashboard if not authenticated
- restrict the user from the login page if they are logged in
#### How should this be manually tested?
- clone the repository'
- cd into the project directory
- run `npm install`
- run `npm start` to start the development server
- attempt to hit the `/dashboard` route without logging in. This should return you to the root route `/`, which is the route to the login page.
- sign in, then attempt to go back to the login page. This should keep you at the dashboard page.
#### Any background context you want to provide?
The local server runs at `http://localhost:3000`
#### What are the relevant pivotal tracker stories?
#6 
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a
